src/renderer/library/components/dialog/Information.tsx

```diff
@@ -34,45 +39,107 @@ interface IProps extends IBaseProps, ReturnType<typeof mapDispatchToProps>, Retu
 }
 
 export class Information extends React.Component<IProps, undefined> {
-    private parsedMarkdown: string;
+
+    private manifestView: PublicationView;
 
     constructor(props: IProps) {
         super(props);
     }
 
     public async componentDidMount() {
-        const { locale } = this.props;
-        const infoFolderRelativePath = "assets/md/information";
+        const { locale, __ } = this.props;
+        const infoFolderRelativePath = "assets/md/information/"; // final / is important
+        const imagesFolder = "images";
+
+        let aboutLocale = locale;
 
-        let folderPath: string = path.join((global as any).__dirname, infoFolderRelativePath);
         try {
+
+            let title = `_______________ ${aboutLocale}`;
+            let [pubView] = await apiAction("publication/search", title);
+            if (pubView) {
+
+                console.log("pubView already exist no need to generate a new one");
+                console.log(pubView);
+
+                this.manifestView = pubView;
+                return;
+            }
+
+            let folderPath = path.join((global as any).__dirname, infoFolderRelativePath);
             if (_PACKAGING === "0") {
                 folderPath = path.join(process.cwd(), "dist", infoFolderRelativePath);
             }
-            const fileContent = await promisify(readFile)(path.join(folderPath,
-                `${locale.toLowerCase()}.md`), {encoding: "utf8"});
-            this.parsedMarkdown = (new commonmark.HtmlRenderer()).render((new commonmark.Parser()).parse(fileContent));
-        } catch (__) {
-            this.parsedMarkdown = "<h1>There is no information for your language</h1>";
+            const filePath = path.join(folderPath, `${aboutLocale}.html`);
+            let htmlFile = `${aboutLocale}.html`;
+            aboutLocale = existsSync(filePath) === true ? locale : "en";
+            htmlFile = `${aboutLocale}.html`;
+            title = `_______________ ${aboutLocale}`;
+
+            [pubView] = await apiAction("publication/search", title);
+            if (pubView) {
+
+                console.log("pubView already exist no need to generate a new one");
+                console.log(pubView);
+
+                this.manifestView = pubView;
+                return;
+            }
+
+            const publication = new R2Publication();
+            publication.Context = ["https://readium.org/webpub-manifest/context.jsonld"];
+            publication.Metadata = new Metadata();
+            publication.Metadata.Title = `${__("reader.footerInfo.moreInfo")} ${aboutLocale}`;
+
+            const link = new Link();
+            link.Href = htmlFile;
+            link.TypeLink = mimeTypes.html;
```

Comment:
Please use XHTML, not HTML. See:
https://github.com/edrlab/thorium-reader/pull/1199/commits/530c537a91d1cf82b4695ef6af78f17b939091b1